changelog.md

```diff
@@ -27,10 +27,13 @@
   It was an oversight to be included in v1.0.
 - `options` now treats `proc` like other pointer types, meaning `nil` proc variables
   are converted to `None`.
+- `joinPath(a,b)` now honors trailing slashes in `b` (or `a` if `b` = "")
 - `relativePath("foo", "foo")` is now `"."`, not `""`, as `""` means invalid path
   and shouldn't be conflated with `"."`; use -d:nimOldRelativePathBehavior to restore the old
   behavior
-- `joinPath(a,b)` now honors trailing slashes in `b` (or `a` if `b` = "")
+- `relativePath(rel, abs)` and `relativePath(abs, rel)` used to silently give wrong results
+  (see #13222); instead they now use `getCurrentDir` to resolve those cases,
+  and this can now throw in edge cases where `getCurrentDir` throws.
```

Comment:
Well getCurrentDir shouldn't raise then.

Reply:
I don't see how that's possible: raiseOSError can be triggered inside `getCurrentDir` (for reasons unrelated to this PR); and it's not theoretical but very real and happened to me a few times eg:
```
mkdir /tmp/d23 && cd /tmp/d23
rmdir /tmp/d23
echo 'import os; echo getCurrentDir()' | nim c -r -f -
/Users/timothee/git_clone/nim/Nim_devel/lib/pure/includes/oserr.nim(94) raiseOSError
Error: unhandled exception: No such file or directory [OSError]
```